include/srf/version.hpp

```diff
@@ -0,0 +1,20 @@
+/*
```

Comment:
This file shouldn't be checked in, it should just be created each time we compile the build.